x/feegrant/keeper/msg_server_test.go

```diff
@@ -0,0 +1,233 @@
+package keeper_test
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/feegrant/keeper"
+
+	"github.com/cosmos/cosmos-sdk/x/feegrant/types"
+)
+
+func (suite *KeeperTestSuite) TestGrantFeeAllowance() {
+	ctx := suite.ctx
+	wrapCtx := sdk.WrapSDKContext(ctx)
+	k := suite.app.FeeGrantKeeper
+	impl := keeper.NewMsgServerImpl(k)
+	atoms := sdk.NewCoins(sdk.NewCoin("atom", sdk.NewInt(1000)))
```

Comment:
Can we add a helper func for these lines? looks like redundant in both tests of this file.